chcemvediet/apps/anonymization/cron.py

```diff
@@ -62,6 +64,31 @@ def normalize_using_libreoffice(attachment):
                           u'unexpected error occured: {}\n{}'.format(
                                   attachment, e.__class__.__name__, trace))
 
+def normalize_using_imagemagic(attachment):
+    try:
+        with Image(filename=attachment.file.path) as original:
+            with original.convert(u'pdf') as converted:
+                with tempfile.NamedTemporaryFile() as temp:
+                    converted.save(filename=temp.name)
```

Comment:
Naozaj potrebujes temp file? Co tak to rovno z `converted` precitat do binerneho stringu: http://docs.wand-py.org/en/0.5.5/guide/write.html#get-binary-string

Reply:
:ok: